packages/components/Display.tsx

```diff
@@ -1,15 +1,28 @@
 import { Flex, Grid } from '@chakra-ui/react';
+import CandidatePhotos from './CadidatePhotos';
 
 import PartyInfo from './PartyInfo';
 
 export default function Display() {
+  const votingScreenColor = '#D9D9D9';
   return (
     <Grid templateColumns="1fr 1fr" h="80%" alignSelf="center">
```

Comment:
pq n colocar a propriedade `bg` aq ja q é o mesmo valor para as childrens aq

Reply:
Como to usando so local nao compensa, porque isso vai ser refatarado pelo theme do chakra, entao nao compensa esforcar muito aqui